packages/web-components/src/components/va-table/va-table-inner/va-table-inner.tsx

```diff
@@ -298,6 +316,13 @@ export class VaTableInner {
     }
   }
 
+  componentWillRender() {
+    // If there are right-aligned columns, split and format them and save in memory for performance reasons
+    if (this.rightAlignCols !== undefined) {
+      this.colsToAlign = this.rightAlignCols.split(',').map(val => Number(val));
```

Comment:
What are your thoughts on adding some type of `console.warn` here if whatever is passed in isn't correct? I don't feel particularly strongly, but I wonder if it'd be useful :thinking: something like "hey right-align-cols expects a string of comma-separated, 0 based, array indexes"

Reply:
Added a warning message, good idea.